Doc/library/bisect.rst

```diff
@@ -140,7 +140,7 @@ thoughts in mind:
 Searching Sorted Lists
 ----------------------
 
-The above :func:`bisect` functions are useful for finding insertion points but
+The above :func:`.bisect` functions are useful for finding insertion points but
```

Comment:
This one I'm not 100% sure about, because the sentence isn't talking about any one `bisect` function in particular. In some ways, the current behaviour is nice — the link takes you to the top of the page, which is the beginning of the section that discusses all the relevant functions

Reply:
I think the intended behaviour is quite clear though as the `:func:` role is used. To link to the overview of both roles, linking to the section would be better I think.

A

Reply:
> To link to the overview of both roles, linking to the section would be better I think.

That sounds like a good idea to me!